02week/blog/index.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+<link rel="stylesheet" href="css/style.css"/>
+  <body>
+    <header>BLOG</header>
+      <main></main>
```

Comment:
The indentation is not correct.
Ideally every blog item would be in its own section, with 3 pieces of content: title, date, and content.